api_fhir_r4/serializers/coverageEligibilityRequestSerializer.py

```diff
@@ -12,7 +12,7 @@ class CoverageEligibilityRequestSerializer(BaseFHIRSerializer):
     logger = logging.getLogger(__name__)
 
     def create(self, validated_data):
-        eligibility_request = EligibilityRequest(uuid=validated_data.get('uuid'),
+        eligibility_request = EligibilityRequest(chf_id=validated_data.get('uuid'),
```

Comment:
@davismutinda how uuid (32 hex) can match a CHF_ID (11 Num) ?

Reply:
@delcroip  this is an example of the payload which is being used 
{
    "resourceType": "CoverageEligibilityRequest",
    "patient":{
        "reference":"Patient/111111117"
    } 
}
the valu being returned as uuid is chf_id

Reply:
or does openMRS use the same uuid stored in openIMIS database for CoverageEligibilityRequest if that is the case i can revert and use uuid